src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -113,6 +119,14 @@ def _type(value):
     return params
 
 
+def enum_default(resource_type, enum_name, enum_val_name):
+    mod = get_versioned_models(resource_type, enum_name)
+    try:
+        return getattr(mod, enum_val_name).value
+    except AttributeError:
+        return None
```

Comment:
That's fine--my main question is, does this introduce a ton of spam to --debug?  I guess it wouldn't if you were mainly developing against latest, in which case, it would be useful.